php/sync-with-algolia/src/index.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+require(__DIR__ . '/../vendor/autoload.php');
+
+use Appwrite\Client;
+use Appwrite\Services\Databases;
+use Appwrite\Query;
+use Algolia\AlgoliaSearch\SearchClient;
+
+return function ($context) {
+    throw_if_missing($_ENV, [
+        'APPWRITE_API_KEY',
+        'APPWRITE_DATABASE_ID',
+        'APPWRITE_COLLECTION_ID',
+        'ALGOLIA_APP_ID',
+        'ALGOLIA_INDEX_ID',
+        'ALGOLIA_ADMIN_API_KEY',
+        'ALGOLIA_SEARCH_API_KEY'
+    ]);
+
+    if ($context->req->method === 'GET') {
+        $html = interpolate(get_static_file('index.html'), [
+            'ALGOLIA_APP_ID' => $_ENV['ALGOLIA_APP_ID'],
+            'ALGOLIA_INDEX_ID' => $_ENV['ALGOLIA_INDEX_ID'],
+            'ALGOLIA_SEARCH_API_KEY' => $_ENV['ALGOLIA_SEARCH_API_KEY'],
+        ]);
+        return $context->res->send($html, 200, [
+            'Content-Type' => 'text/html',
+        ]);
+    }
+
+    $client = new Client();
+    $client
+        ->setEndpoint('https://cloud.appwrite.io/v1')
+        ->setProject(getenv('APPWRITE_FUNCTION_PROJECT_ID'))
+        ->setKey(getenv('APPWRITE_API_KEY'));
```

Comment:
Notice you use `$_ENV` and also `getenv`, lets stick to just one (unless its nessessary to do it in 2 days)